### ISSUE TITLE
Kopernicus doesn't load in planetary systems, specifically, Parallax 2.0.

### ISSUE THREAD

**Issue by shieemane** [Kopernicus/Kopernicus#537]
Had no issues with pre-2.0 parallax, or rather kopernicus before. I removed every mod except the required mods for parallax and kopernicus, still nothing. When loading a new world, the background nature sound plays while planet loading symbol plays uninterrupted. The game plays normally when kopernicus is removed (and thereby parallax 2.0 is also removed/doesnt get loaded) with or without my other mods.
(first time submitting anything on github)
[KSP.log](https://github.com/Kopernicus/Kopernicus/files/9543128/KSP.log)
[Kopernicus.log](https://github.com/Kopernicus/Kopernicus/files/9543129/Kopernicus.log)
[ModuleManagerConfig.txt](https://github.com/Kopernicus/Kopernicus/files/9543141/ModuleManagerConfig.txt)

**Comment by R-T-B** [comment]
You did a good job with the bug report, so no worries there.

The issue I'm seeing is that parallax seems to be constantly having issues launching with users with less than 8GBs of VRAM, like you.  I'm in touch with the author and seeing if we can't do anything about this, but until then, I can only say "try it without parallax" or "get more VRAM video card" which are both very unhelpful, I realize.

I'd just wait it out, honestly.  There is likely to be a bugfix on parallax's side soon.

**Comment by R-T-B** [comment]
After discussion with GamesLinx, it seemed I jumped to conclusions a bit there.  It's not a VRAM issue.

You do however have what appears to be a broken parallax install, including the whole source directory "tesselation" in your gamedata, rather than the release folder.

Can you try starting from scratch and just installing the bare essentials, adding in components one by one?

**Comment by shieemane** [comment]
> You do however have what appears to be a broken parallax install, including the whole source directory "tesselation" in your gamedata, rather than the release folder. I'd certainly not want that in there.

About that, I originally was downloading parallax from CKAN, however it it kept failing to install. I tried 2 more times with it failing in the installation phase (files got downloaded but not installed). I then downloaded it manually (thinking tessellation was also required) but seems like I didnt install correctly, I believe I didnt get any kopernicus warning either when launching (but I might be wrong).
(according to the inst. instructions for parallax, you must extract the 2 texture files and then merge into one, however it seems to merge automatically, as I only get 1 file when extracting them both)

After that, I tried to download it from CKAN with the manually installed files still there and overwriting them in CKAN, this time though, it required me to install scatterer 3 with an obligatory sunflare mod, this time the installation didnt fail. I launched but still the same as before.

Now after your advice, I have deleted everything and re-installed. It worked. The manually downloaded files must have been broken or the Tessellation file might have interfered... Im guessing this might be something on CKAN, as it didnt require me to download scatterer 3 with a sunflare mod of my choice the first 3 times (I restarted the program and PC the 3rd time)

Personally have no idea what I did that is different from the first 3 times, however hopefully this info comes of use to other people having the same issue. 

Thank you and Linx for the product and help, you both are great developers!
